test-conf/proxy.conf

```diff
@@ -0,0 +1,378 @@
+#
```

Comment:
We need them here.
https://github.com/apache/pulsar-client-cpp/pull/389/files#diff-112cae152238b3448927abaa1c29664a704289bdfab104d9ba478565aa9cc822R86

https://github.com/apache/pulsar-client-cpp/pull/389/files#diff-112cae152238b3448927abaa1c29664a704289bdfab104d9ba478565aa9cc822R150

Reply:
```
command: bash -c "bin/apply-config-from-env.py conf/proxy.conf && exec bin/pulsar proxy"
```
This command is used to apply config from env to container's `conf/proxy.conf`. It will not use outside configuration.

Reply:
oh. I thought these configs are copied to the containers, but seem not. Let me know if we want to remove this config.

Reply:
If these configs are not used, we can remove them.

Reply:
Removed.